docs/rules/ultiduo.md

```diff
@@ -38,10 +47,50 @@ For playoff matches following a best of three (BO3) format, the **higher seeded
 
 The **higher seeded team** is **Team A**. **Maps must be played in the order in which they were picked.**
 
+Using the Captain Discord Match Thread channel 
 1. **Team A bans** a map
 2. **Team B bans** a map
 3. **Team A picks** a map
 4. **Team B picks** a map
 5. **Team A bans** a map
 6. **Team B bans** a map
-7. **Team A picks** a map
\ No newline at end of file
+7. **Team A picks** a map
+
+## 3. Pausing
+Pauses during a match are allowed for any reason, except for immediately after readying up with the intent of delaying the match.
+
+The match may only be unpaused by the team who initiated the pause, with confirmation from the opposing teams. Unpausing initiates a five-second countdown.
+
+### 3.1. Pause Limits
+Each team is allowed two pauses per match, with a combined total time limit of five minutes. 
+
+### 3.2. Tactical Pauses
+Tactical pauses are allowed but contribute to the pause limits described above. They should not be used to disrupt ongoing plays.
+
+### 3.3. Pause Command
+To pause a match, teams must open their console and send the command: pause. The same command can be sent to unpause the match.
+
+### 3.4. Repause Command
+To allow a player to rejoin the server during a pause, use the command: repause. No warning is required to use the repause command.
+
+### 3.5. Abuse of Pause function
+If there are any complications or a team is breaching any of the pause rules, a tournament administration must be contacted **immediately.**
+
+## 4. Player Eligibility
+### 4.1. Ping Requirements //make this not 3 to avoid crossing over with global ruleset numbering conflictions
+ Ultiduo will not follow our Global Ruleset Ping or Out of Region rules. Out-of-Region Players intending to join a roster, or play as a mercenary, must meet the in-game ping limit of 120. If you are unsure whether you meet this requirement, prior to the tournament starting, you should contact a League Administrator to undergo a ping check. It is the responsibility of Out-of-Region Players to ensure that they satisfy this ping limit before the beginning of the tournament. 
```

Comment:
"prior to the tournament starting, you should contact a League Administrator to undergo a ping check.", remove the comma between starting and you should

"prior to the tournament starting you should contact a League Administrator to undergo a ping check."

Also, ping limit should be at least 150.